public class ExceptionExample2 {
    public static void main(String[] args) {
        System.out.println("Reset of the code....");

        try
        {
            System.out.println("in try block");
            int a=10/0;
        }//try end
    }
    
}
